src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -6,53 +6,73 @@ import {
   unstable_JrRestApi,
 } from 'scrivito'
 import personCircle from '../../assets/images/person-circle.svg'
+import { ensureString } from '../../utils/ensureString'
 
 export const CurrentUser = provideDataItem('CurrentUser', {
   async get() {
     const user = await load(currentUser)
     if (!user) return {}
 
-    const rawNeoletterData = await unstable_JrRestApi.get(neoletterProfileUrl())
-
-    let company: string | undefined
-    let phoneNumber: string | undefined
-    let salutation: string | undefined
-    const pisaUserId: string | undefined = 'F87BDC400E41D630E030A8C00D01158A'
-    const picture: string = user.picture() || personCircle
-
-    if (isNeoletterData(rawNeoletterData)) {
-      company = rawNeoletterData.company || ''
-      phoneNumber = rawNeoletterData.phone_number || ''
-      salutation = rawNeoletterData.salutation || ''
+    const neoletterProfile = await unstable_JrRestApi.get(neoletterProfileUrl())
+    if (!isNeoletterData(neoletterProfile)) {
+      throw new Error('Neoletter data is not in the expected format')
     }
 
+    // TODO: read data from Pisa
+    const pisaUserId: string = 'F87BDC400E41D630E030A8C00D01158A'
+    const salesUserId: string = '052601BEBCEC39C8E040A8C00D0107AC'
+    const serviceUserId: string = 'D456ACF6FF405922E030A8C02A010C68'
+
     return {
-      company,
       email: user.email(),
+      picture: ensureString(user.picture()) || personCircle,
       jrUserId: user.id(),
+
       pisaUserId,
-      name: user.name(),
-      phoneNumber,
-      picture,
-      salesUserId: '052601BEBCEC39C8E040A8C00D0107AC', // TODO: replace with real Pisa data
-      salutation,
-      serviceUserId: 'D456ACF6FF405922E030A8C02A010C68', // TODO: replace with real Pisa data
+      salesUserId,
+      serviceUserId,
```

Comment:
* [ ] salesUserId and serviceUserId are not part of provideEditingConfig(CurrentUser). It's not clear why they are here.

Reply:
Good catch! They are currently hard-coded in `PersonCardWidget` but we might switch this over to a datalocator.